content/22.vaccines-trad.md

```diff
@@ -394,66 +395,68 @@ Because the Covifenz results became available relatively recently (May 4, 2022),
 Their intention is to accelerate the development of COVID-19 vaccines, diagnostics, and therapeutics and to ensure the equitable distribution of vaccines to low- and middle-income countries [@doi:10.1111/1468-0009.12503; @url:https://www.who.int/publications/m/item/fair-allocation-mechanism-for-covid-19-vaccines-through-the-covax-facility].
-COVAX invested in several vaccine programs to ensure they would have access to successful vaccine candidates [@doi:10.1126/science.369.6503.489]. However, the initiative has been less successful than was initially hoped.
+COVAX invested in several vaccine programs to ensure they would have access to successful vaccine candidates [@doi:10.1126/science.369.6503.489].
+However, the initiative has been less successful than was initially hoped. <!-- In what way? The inequities described below? -->
 
 Additionally, the vaccine technologies available differ widely around the globe.
 As we review elsewhere [@individual-vaccines-novel], wealthier nations have invested heavily in mRNA and DNA vaccines.
 In contrast, as we describe above, many countries outside of Europe and North America have developed highly effective vaccines using more traditional approaches.
-As a result, there is a clear relationship between a country's gross domestic product (GDP) and its access to vaccines.
+As a result, there is a clear relationship between a country's gross domestic product (GDP) and its access to vaccines (Figure @fig:gdp-scatter).
 
 ![
 **Vaccine Distribution across Platform Types as a Function of GDP.**
 As of {{owid_most_recent_date}}, the number of doses distributed, by platform type, is shown as a function of GDP.
 These data are retrieved from Our World in Data [@url:https://ourworldindata.org/coronavirus; @doi:10.1038/s41562-021-01122-8] and plotted using the Python package plotnine [@url:https://github.com/has2k1/plotnine].
 Lines show a general trend in the data and are drawn using geom_smooth [@url:https://ggplot2.tidyverse.org/reference/geom_smooth.html].
-See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated every time the manuscript is.
+See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated every time the manuscript is regenerated. <!-- In some places we say daily. Should we be more vague? The reality is complex because the figure is daily but the HTML is less regular. -->
 Axes are not scaled per capita because both variables would be expected to be modulated by population size.
 ]({{doses_scatterplot}} "Distribution of vaccines according to GDP"){#fig:gdp-scatter secno=1}
 
+<!-- Figure is not rendering -->
```

Comment:
@agitter Did you review a version you built or did you review the one that is currently being built here? Because I don't think the `output` branch has updated since #1150 was merged in.

Reply:
I was reading through the July 26 GitHub pages version.  There have been a lot of changes since then, so I expect this has been resolved.  I wanted to make a new, merge in all your updates, and then remove this comment after confirming the figure was fixed.